snippets/javascript/class-tag-on-scroll.md

```diff
@@ -0,0 +1,81 @@
+# change class tag on scroll
+
+## description
+
+> this snippet will change a preassigned tag to a class on scroll by adding an
+> 'event listener' property
+
+## application
+
+> this snippet is useful when we want to activate an animation or reveal
+> elements on scroll
+
+### code
+
+```javascript
+"use strict";
+
+window.addEventListener("scroll", show);
+
+/* ^ EventListener is the name of the property. - show-  will be the name
+of the second tag */
```

Comment:
Almost! 🙂 addEventListener is a **function** with two parameters:
1. the **name of the event** as a string
2. the name of a **function that needs to be executed** when the event is triggered